Microsoft.Azure.Functions.PowerShellWorker.Common.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <BuildNumber Condition="$(APPVEYOR) != ''">$(APPVEYOR_BUILD_NUMBER)</BuildNumber>
+    <BuildNumber Condition="$(APPVEYOR) == ''">9999</BuildNumber>
+    <Version>0.1.$(BuildNumber)-alpha</Version>
```

Comment:
Seems like we should call this `preview` instead of `alpha`.

Reply:
changed to preview